revscoring/features/diff.py

```diff
@@ -39,96 +42,106 @@ def process_chars_removed(diff_removed_segments):
                         returns=int, depends_on=[diff.removed_segments])
 
 proportion_of_chars_removed = chars_removed / \
-              modifiers.max(parent_revision.chars, 1)
+    modifiers.max(parent_revision.chars, 1)
```

Comment:
This is supposed to be tabbed in at least two levels.  
https://www.python.org/dev/peps/pep-0008/#indentation